spec/facility_spec.rb

```diff
@@ -17,10 +19,190 @@
   describe '#add service' do
     it 'can add available services' do
       expect(@facility.services).to eq([])
+
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
       @facility.add_service('Vehicle Registration')
+
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#has service?' do
+    it 'can check if a facility has a service' do
+      expect(@facility.has_service?('Vehicle Registration')).to eq(false)
+
+      @facility.add_service('Vehicle Registration')
+
+      expect(@facility.has_service?('Vehicle Registration')).to eq(true)
+
+    end
+  end
+
+  describe '#allow_service' do
+    it 'allows a service that has been added' do
+      @facility.add_service('Vehicle Registration')
+      @facility.has_service?('Vehicle Registration')
+
+      expect(@facility.allow_service('Vehicle Registration'){'Registered'}).to eq('Registered')     
+    end
+  end
+
+  describe '#allow_service' do
+    it 'doesnt allow a service that hasnt been added' do
+      expect(@facility.allow_service('Vehicle Registration'){'Registered'}).to eq(false)     
+    end
+  end
+
+  describe '#registered vehicles' do
+    it 'can list vehicles that have been registered at a facility' do
```

Comment:
Based on your it statement, I would think this test is holding assertions showing us adding vehicles. Maybe instead this it block could say `it "starts with no vehicles registered"`. We want to be specific and explicit about what the it block is demonstrating.